### ISSUE TITLE
Failing Tests display twice in HTML

### ISSUE THREAD

**Issue by sgnl** [mochajs/mocha#2166]
Here is a link to the [jsfiddle](https://jsfiddle.net/sgnl/wfyavarf/2/).

Passing tests correctly pass and display only once.

**Comment by danielstjules** [comment]
Thanks for letting us know! This is a duplicate of https://github.com/mochajs/mocha/issues/2083 and has will be fixed in https://github.com/mochajs/mocha/pull/2112

**Comment by sgnl** [comment]
TY. sorry for not finding the dupe issue. cheers!
